src/rdkafka_request.h

```diff
@@ -107,6 +107,55 @@ typedef struct rd_kafkap_NodeEndpoints_s {
         rd_kafkap_NodeEndpoint_t *NodeEndpoints;
 } rd_kafkap_NodeEndpoints_t;
 
+/**@}*/
+/**
+ * @name Produce tags
+ * @{
+ *
+ */
+
+typedef struct rd_kafkap_Produce_reply_tags_Partition_s {
+        int32_t Partition;
+        rd_kafkap_CurrentLeader_t CurrentLeader;
+} rd_kafkap_Produce_reply_tags_Partition_t;
+
+typedef struct rd_kafkap_Produce_reply_tags_Topic_s {
+        char *TopicName;
+        rd_kafkap_Produce_reply_tags_Partition_t *Partition;
+} rd_kafkap_Produce_reply_tags_Topic_t;
+
+typedef struct rd_kafkap_Produce_reply_tags_s {
+        int32_t LeaderChangeCnt;
+        rd_kafkap_NodeEndpoints_t NodeEndpoints;
+        rd_kafkap_Produce_reply_tags_Topic_t *Topic;
+} rd_kafkap_Produce_reply_tags_t;
+
+/**@}*/
+/**
+ * @name Fetch tags
+ * @{
+ *
+ */
+
+typedef struct rd_kafkap_Fetch_reply_Partition_s {
```

Comment:
```suggestion
typedef struct rd_kafkap_Fetch_reply_Partition_s {
```

Reply:
Updated to rd_kafkap_Fetch_reply_tags_Partition_s